### ISSUE TITLE
module 'openai' has no attribute 'Audio'

### ISSUE THREAD

**Issue by agarzon-orangerine** [karfly/chatgpt_telegram_bot#388]
Since the latest update, commit ce0c34825f91afe865ed047acbfea542e47c9173

```
 Exception while handling an update:                                                                                          │
│   Traceback (most recent call last):                                                                                           │
│     File "/usr/local/lib/python3.8/site-packages/telegram/ext/_application.py", line 1104, in process_update                   │
│       await coroutine                                                                                                          │
│     File "/usr/local/lib/python3.8/site-packages/telegram/ext/_handler.py", line 141, in handle_update                         │
│       return await self.callback(update, context)                                                                              │
│     File "bot/bot.py", line 350, in voice_message_handle                                                                       │
│       transcribed_text = await openai_utils.transcribe_audio(buf)                                                              │
│     File "/code/bot/openai_utils.py", line 193, in transcribe_audio                                                            │
│       r = await openai.Audio.atranscribe("whisper-1", audio_file)                                                              │
│   AttributeError: module 'openai' has no attribute 'Audio' 
```

I'm using the docker version.

**Comment by karfly** [comment]
Just fixed openai version in [977ffb5](https://github.com/karfly/chatgpt_telegram_bot/commit/977ffb5e323bc906fdf57e51c01ae235c8b47731). Please, try again

**Comment by agarzon-orangerine** [comment]
It works now 👍🏻
